vectordatasource/meta/sql.py

```diff
@@ -169,6 +169,7 @@
     'mz_calculate_ferry_level':           'mz_calculate_ferry_level',
     'util.is_building':                   'mz_calculate_is_building_or_part',
     'util.tag_str_to_bool':               'notimplemented',
+    'util.safe_int':                      'notimplemented',
```

Comment:
It's a function which doesn't exist, and will complain about `notimplemented` not existing if anyone tries to calculate the properties in SQL rather than Python. It's on a code path that we don't use any more and might get rid of in the future.